### PR TITLE
pacific: quincy feature, custom git_url for cephadm_branch

### PR DESCRIPTION
**Pull request by liewegas**
These are the pacific-relevant patches from #39039

### CODE REVIEW

**Review by jdurgin — APPROVED** [review]

File: src/include/ceph_features.h [hunk 1]
```diff
@@ -248,6 +248,7 @@ DEFINE_CEPH_FEATURE_DEPRECATED(63, 1, RESERVED_BROKEN, LUMINOUS) // client-facin
 	 CEPH_FEATUREMASK_OSD_REPOP_MLCOD | \
 	 CEPH_FEATUREMASK_SERVER_PACIFIC | \
 	 CEPH_FEATURE_OSD_FIXED_COLLECTION_LIST | \
+	 CEPH_FEATUREMASK_SERVER_QUINCY | \
```
Comment (jdurgin) [inline src/include/ceph_features.h:251]: do we need this in pacific?

**Comment by liewegas** [comment]
I think on occasion we've wanted to test for future releases?  Maybe not.
In any case, I thought it might avoid a future conflict.

>

**Comment by jdurgin** [comment]
> I think on occasion we've wanted to test for future releases? Maybe not. In any case, I thought it might avoid a future conflict.
> […](#)

adding the feature bit is ok, it's the inclusion in the ALL mask that I wasn't sure of - I thought we used that for advertised features/encoding in some places

**Comment by neha-ojha** [comment]
> > I think on occasion we've wanted to test for future releases? Maybe not. In any case, I thought it might avoid a future conflict.
> > […](#)
> 
> adding the feature bit is ok, it's the inclusion in the ALL mask that I wasn't sure of - I thought we used that for advertised features/encoding in some places

looks like we've done neither for previous releases

**Comment by liewegas** [comment]
You're both right :), dropping the commit

**Review by neha-ojha — APPROVED** [review]

**Comment by neha-ojha** [comment]
jenkins test make check

**Review by sebastian-philipp — APPROVED** [review]

**Comment by sebastian-philipp** [comment]
cepham tests passed:  https://pulpito.ceph.com/swagner-2021-02-09_10:28:14-rados:cephadm-wip-swagner2-testing-2021-02-08-1109-pacific-distro-basic-smithi/

Failures: 

* https://tracker.ceph.com/issues/49232
* https://tracker.ceph.com/issues/49233

**Pull request merged**